.github/scripts/unit_tests_script.sh

```diff
@@ -0,0 +1,77 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
+
+set -e
+
+unset _JAVA_OPTIONS
+
+# Set MAVEN_OPTS for Surefire launcher. Skip remoteresources to avoid intermittent connection timeouts when
```

Comment:
Skipping remote dependencies is odd. If we've cached all the required dependencies, then Maven should not be going upstream. If we've not cached, then the build will fail if it can't go upstream.

So, I wonder if we've looked into the issue we're trying to solve: *why* was maven trying to hit SIGAR? Why are things not cached? And, if they are cached, why isn't Maven using them?

Reply:
Hi @paul-rogers, Thanks for pointing this out, Updated PR, and turns out it isn't needed.